packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/ImageEdit.ts

```diff
@@ -671,3 +672,10 @@ function isASmallImage(editInfo: ImageEditInfo, isFeatureEnabled?: boolean) {
     const { widthPx, heightPx } = editInfo;
     return widthPx && heightPx && widthPx * widthPx < MAX_SMALL_SIZE_IMAGE && isFeatureEnabled;
 }
+
+function getColorString(color: string | ModeIndependentColor, isDarkMode: boolean): string {
+    if (typeof color === 'string') {
+        return color.trim();
+    }
+    return isDarkMode ? color.darkModeColor.trim() : color.lightModeColor.trim();
```

Comment:
What if the color is undefined?

Reply:
@haven2world would you please take a look at my comment? thanks.

Reply:
I replied but I don't know why you cannot see it
<img width="807" alt="image" src="https://user-images.githubusercontent.com/17299418/173474836-d5fd64bb-38e6-4b48-8e7a-f15228bc5ab3.png">

Reply:
Sorry, looks like I didn't commit the comment... I'm not familiar with the workflow on GitHub. Haha it's kind of different from ADO